diff --git a/command.py b/command.py
index aac002e..046ab56 100644
--- a/command.py
+++ b/command.py
@@ -1,2 +1,2 @@
 def command_for_revision(revision: str):
-    return ["git", "show", revision]
+    return f"git show {revision}"
diff --git a/runner.py b/runner.py
index 2ccb144..fdef0b4 100644
--- a/runner.py
+++ b/runner.py
@@ -5,4 +5,4 @@ from command import command_for_revision
 def show_revision(revision: str):
     if os.name != "posix":
         raise RuntimeError("This command runner requires POSIX")
-    return subprocess.run(command_for_revision(revision), check=True, capture_output=True)
+    return subprocess.run(command_for_revision(revision), shell=True, check=True, capture_output=True)
